test/jdk/java/foreign/TestHFA.java

```diff
@@ -50,7 +50,7 @@ public class TestHFA {
     final static Linker abi = Linker.nativeLinker();
     final static SymbolLookup lookup = SymbolLookup.loaderLookup();
 
-    static final OfFloat FLOAT = JAVA_FLOAT.withBitAlignment(32);
+    static final OfFloat FLOAT = JAVA_FLOAT.withByteAlignment(4);
```

Comment:
Is this even required - e.g. the alignment of Java float is 4 bytes...

Reply:
It's probably not required. It was derived from a jextract generated file.

Reply:
Do you prefer removing it?

Reply:
I think it's not important. I'll just integrate it to get the tests working again.

Reply:
Yeah, let's integrate